src/repr/src/adt/regex.rs

```diff
@@ -51,60 +51,60 @@ include!(concat!(env!("OUT_DIR"), "/mz_repr.adt.regex.rs"));
 /// (Earlier, serialization was buggy due to <https://github.com/tailhook/serde-regex/issues/14>,
 /// and also making the same mistake in our own protobuf serialization code.)
 #[derive(Debug, Clone, MzReflect)]
-pub struct Regex {
+pub struct RegexWrapper {
```

Comment:
As discussed in Slack, I'd prefer not doing this rename because it makes the usage sites slightly harder to read and might prompt people to concern themselves with the details of this type even when they don't need to.

Reply:
Reverted